flow/templates/print_status.sh

```diff
@@ -0,0 +1,55 @@
+{# The following variables are available to all scripts. #}
```

Comment:
Since the status view is not a script, I propose to use the file suffix `.jinja` or `.txt` instead of `.sh`.